controllers/replication/secret_controller_test.go

```diff
@@ -456,44 +453,3 @@ func TestRequiresUpdate(t *testing.T) {
 	syncSecrets(orig, dest)
 	assert.False(requiresUpdate(orig, dest))
 }
-
-func managedReplicatedSecret(t *testing.T, ctx context.Context, f *framework.Framework, namespace string) {
```

Comment:
What was the problem with this test?

Reply:
The test was verifying behavior in K8ssandaClusterReconciler. The verifications are now done in k8ssandracluster_controller_test.go.